test/integration/__snapshots__/integration.test.ts.snap

```diff
@@ -429,3 +429,44 @@ export const tables = {
 }
 "
 `;
+
+exports[`Integration tests Schema generation prefixWithSchemaNames generation 1`] = `
+"/* tslint:disable */
+/* eslint-disable */
+
+
+export type Json = unknown;
+export type CommentStatusEnum = 'archived' | 'complete' | 'deleted' | 'pending';
+
+// Table public_Users
+export interface PublicUsers {
+  id: string;
+  name: string;
+  pronoun: string | null;
+}
+export interface PublicUsersInput {
+  id?: string;
+  name: string;
+  pronoun?: string | null;
+}
+const public_Users = {
+  tableName: 'public.Users',
```

Comment:
This should be `public.users` (lowercase `u`). (My suggestion above should fix this.)